normandy/control/templates/control/admin/login.html

```diff
@@ -13,24 +17,26 @@
     {% endfor %}
   {% endif %}
 
-  <div id="content-main">
-    <form action="{{ app_path }}" method="post" id="login-form">{% csrf_token %}
-      <div class="form-row">
+  <form action="{{ app_path }}" method="post" id="login-form">{% csrf_token %}
+    <div class="row">
+      <div class="fluid-8">
         {{ form.username.errors }}
-        {{ form.username.label_tag }} {{ form.username }}
+        {{ form.username.label_tag }} <div class="input-with-icon user">{{ form.username }}</div>
       </div>
-      <div class="form-row">
+    </div>
+    <div class="row">
+      <div class="fluid-8">
         {{ form.password.errors }}
-        {{ form.password.label_tag }} {{ form.password }}
+        {{ form.password.label_tag }} <div class="input-with-icon key">{{ form.password }}</div>
         <input type="hidden" name="next" value="{{ next|default:"/control" }}" />
       </div>
-      <div class="submit-row">
-        <label>&nbsp;</label><input type="submit" value="Log in" />
-      </div>
-    </form>
+    </div>
+    <div class="row">
+      <label>&nbsp;</label><input type="submit" class="button" value="Log in" />
+    </div>
+  </form>
 
-    <script type="text/javascript">
-      document.getElementById('id_username').focus()
-    </script>
-  </div>
+  <script type="text/javascript">
```

Comment:
we could even just have a really generic way of solving this like:

`HTML`
```html
<input type="text" name="whatever" id="id_whatever" data-on-load="focus">
```
`JS: onload.js`
```js
document.addEventListener("DOMContentLoaded", function(event) { 
	var elems = document.querySelectorAll('[data-on-load]');

	for (var i=0; i < elems.length; i++) {
		var elem = elems[i];
		switch (elem.dataset['onLoad']) {
			case 'focus':
				elem.focus();
				break;
		}
	}
});
```